WordPress/Resources/tr.lproj/Localizable.strings

```diff
@@ -1833,6 +1833,10 @@ translators: Block name. %s: The localized block name */
 /* Button title displayed in popup indicating the user copied the local copy */
 "Copy the version from this app" = "Sürümü bu uygulamadan kopyala";
 
+/* Label for page copy link option. Tapping copies of the page URL to clipboard.
+   Label for post copy link option. Tapping copies of the post URL to clipboard. */
+"Copy Link" = "Bağlantıyı kopyala";
```

Comment:
Thanks for helping improve the translations!

Note thought that in practice we handle translations using a separate service — namely using GlotPress and https://translate.wordpress.org/, which means that that change will be overwritten by our tools next time we will re-import the latest translations from GlotPress.

To contribute to translations on WordPress, the right approach is to contribute on https://translate.wordpress.org/.
You can find more info about the process and how to help in [our Translator Handbook](https://make.wordpress.org/polyglots/handbook/translating/glotpress-translate-wordpress-org/), but the TL;DR is that you can create an account here (if you don't have one already) and suggest a Turkish translation for this string on [this GlotPress project](https://translate.wordpress.org/projects/apps/ios/dev/). Thanks!

Reply:
@AliSoftware ,
 I removed this part. Thank you for your help.